modules/src/stocks.py

```diff
@@ -0,0 +1,28 @@
+import requests
+import json
+
+from templates.text import TextTemplate
+
+
+def process(input, entities):
```

Comment:
Can you use any other variable name instead of `input`. It is a Python built-in https://docs.python.org/2/library/functions.html#input

Reply:
gotcha, on it.
Any ideal how to train wit.ai for JARVIS?

Reply:
and pass the tests?

Reply:
@ab9-er Any ideal how to train wit.ai for JARVIS and pass the tests?